runtime/print.c

```diff
@@ -142,6 +143,13 @@ static void printTagged(FILE *fp, TaggedPtr p, int style, struct PrintStack *sta
                 }
             }
             break;
+        case TAG_DECIMAL:
+            {
+                char decStr[DECQUAD_String];
+                decQuadToString((void *)taggedToPtr(p), decStr);
```

Comment:
Shouldn't this be a case to `decQuad *` ?

Reply:
Changed